mkdocs.yml

```diff
@@ -6,22 +6,24 @@ nav:
     - "Quick Start": quick-start.md
     - Tutorial: tutorial.md
     - Examples: examples.md
-    - Reference:
-        - "Wildcard Listeners": wildcard-listeners.md
-        - "Listener Aggregates": aggregates.md
-        - "Lazy Listeners":
-            - Intro: lazy-listeners/intro.md
-            - LazyListener: lazy-listeners/lazy-listener.md
-            - LazyEventListener: lazy-listeners/lazy-event-listener.md
-            - LazyListenerAggregate: lazy-listeners/lazy-listener-aggregate.md
-        - "EventManager API": api.md
-        - "Intercepting Filters": intercepting-filters.md
-        - "Migration Guide":
-            - Intro: migration/intro.md
-            - "Removed Functionality": migration/removed.md
-            - "Changed Functionality": migration/changed.md
+    - "Wildcard Listeners": wildcard-listeners.md
+    - "Listener Aggregates": aggregates.md
+    - "Lazy Listeners":
+        - Introduction: lazy-listeners/intro.md
+        - LazyListener: lazy-listeners/lazy-listener.md
+        - LazyEventListener: lazy-listeners/lazy-event-listener.md
+        - LazyListenerAggregate: lazy-listeners/lazy-listener-aggregate.md
+    - "EventManager API": api.md
+    - "Intercepting Filters": intercepting-filters.md
+    - "Application Integration":
+        - "Usage in a laminas-mvc Application": application-integration/usage-in-a-laminas-mvc-application.md
+    - "Migration Guide":
+        - Introduction: migration/intro.md
+        - "Removed Functionality": migration/removed.md
+        - "Changed Functionality": migration/changed.md
```

Comment:
The promotion of these pages to a higher level in the TOC feels like an unrelated change; could we maybe do this part in a separate PR?

Reply:
I have reverted the changes, but only a depth of two is correctly displayed in the navigation: https://docs.laminas.dev/laminas-eventmanager/